test/simple/simptest.c

```diff
@@ -868,7 +897,8 @@ static pmix_status_t notify_event(pmix_status_t code,
                                   pmix_info_t info[], size_t ninfo,
                                   pmix_op_cbfunc_t cbfunc, void *cbdata)
 {
-    return PMIX_SUCCESS;
+    pmix_output(0, "SERVER: NOTIFY EVENT");
```

Comment:
Should this be verbose only, or did you want it to be always displayed?

Reply:
Oh wait - yeah, I do want that every time. Didn't look closely enough to realize it was in the test.